alignak/http/generic_interface.py

```diff
@@ -224,9 +222,12 @@ def get_external_commands(self):
         :return: serialized external command list
         :rtype: str
         """
-        with self.app.external_commands_lock:
-            cmds = self.app.get_external_commands()
-            raw = serialize(cmds)
+        if hasattr(self.app, 'external_commands_lock'):
```

Comment:
One of the daemons (not sure witch not have this and so prevent error in cherrypy)